common/fe-ste-models.go

```diff
@@ -595,6 +595,8 @@ func (TransferStatus) SkippedFileAlreadyExists() TransferStatus { return Transfe
 
 func (TransferStatus) SkippedBlobHasSnapshots() TransferStatus { return TransferStatus(-4) }
 
+func (TransferStatus) TierAvailabilityCheckFailure() TransferStatus { return TransferStatus(-5) }
```

Comment:
Please add to the changelog a point that says that this new status can be returned (ssince they are visible in azcopy jobs show and, IIRC, in the json format output